x-pack/plugins/lens/public/editor_frame_service/editor_frame/frame_layout.scss

```diff
@@ -88,8 +88,7 @@ a tilemap in an iframe: https://github.com/elastic/kibana/issues/16457 */
   position: relative;
 }
 
-.lnsFrameLayout-isFullscreen .lnsFrameLayout__sidebar--left,
-.lnsFrameLayout-isFullscreen .lnsFrameLayout__suggestionPanel {
+.lnsFrameLayout-isFullscreen .lnsFrameLayout__sidebar--left {
   // Hide the datapanel and suggestions in fullscreen mode. Using display: none does trigger
```

Comment:
Small nit, but might be worth updating this comment to no longer reference suggestions.